### ISSUE TITLE
Error in update method

### ISSUE THREAD

**Issue by silvaneojunior** [silvaneojunior/kDGLM#1]
The results of fitting one observation at a time is different from fitting all observations simultaneously.
Seems to happen only if the initial outcome has only one observation.

Example:

```r
structure=polynomial_block(mu=1,D=c(0.95,0.95),R1=1,order=2)+
  polynomial_block(phi=1,D=0.9,R1=1)

outcome=Gamma(mu='mu',phi='phi',data=series[1:2],alt.method=TRUE)
outcome$update=update_FGamma_alt

model=fit_model(structure,outcome)
```

**Comment by silvaneojunior** [comment]
Fixed in version 1.2.1.

**Comment by silvaneojunior** [comment]
Fixed in version 1.2.1.
